render/image.go

```diff
@@ -54,51 +73,56 @@ func rotateImage(src image.Image, rot rotations) image.Image {
 	return src
 }
 
-func drawWatermark(dc *gg.Context) {
-	watermarkImage, err := media.GetWatermarkPNG(dc.Width()*2/3, dc.Height()*2/3)
+func drawWatermark(dc *boardContext) {
+
```

Comment:
This was updated so it properly sizes the watermark according to the rendered board size, not just the size of the entire image (GIF). This ensures that the watermark stays within the board.